src/unitxt/processors.py

```diff
@@ -215,3 +215,11 @@ def process_value(self, text: Any) -> Any:
         if self.string.lower() in text.lower():
             return self.string.lower()
         return text
+
+
+class ExtractMtBenchJudgment(FieldOperator):
+    def process_value(self, text: Any) -> Any:
+        match = re.search(r"\[\[([\d]+\.?[\d]*)\]\]", text)
```

Comment:
did you verify this re pattern works?

Reply:
I know I wrote it (copied it actually), but lets verify if you didn't

Reply:
also we want to return the reason as well I think

Reply:
I verify. You can see that I added a test for this processor.